src/components/EditWebhooks.js

```diff
@@ -192,44 +255,66 @@ class EditWebhooks extends React.Component {
 
     let submitBtnMessageId = 'save';
     if (['loading', 'saved'].includes(status)) {
-      submitBtnMessageId = status;
+      submitBtnMessageId = 'saved';
     }
     const submitBtnLabel = this.messages[submitBtnMessageId] && intl.formatMessage(this.messages[submitBtnMessageId]);
+    const submitBtnLabelEnd = this.messages['title'] && intl.formatMessage(this.messages['title']);
+    const captializedSubmitBtnLabel = submitBtnLabel.charAt(0).toUpperCase() + submitBtnLabel.slice(1);
+    const webHookCount = this.getWebhookCount();
 
     return loading ? (
       <Loading />
     ) : (
-      <div className="EditWebhooks">
+      <div css={{ 'margin-left': '2.5rem' }} className="EditWebhooks">
         <style jsx>
           {`
             .error {
               color: red;
             }
           `}
         </style>
-
         <div className="webhooks">
-          <h2>{this.props.title}</h2>
+          <h2 className="">{this.props.title}</h2>
+          <StyledHr my={4} />
           {webhooks.map(this.renderWebhook)}
         </div>
 
         <div className="editWebhooksActions">
-          <Button bsStyle="primary" onClick={() => this.addWebhook()}>
+          <StyledButton
+            buttonStyle="standard"
+            buttonSize="medium"
+            onClick={this.addWebhook}
+            css={{ width: '65%' }}
+            borderStyle="dashed"
+            borderWidth="1px"
+            borderRadius={4}
+            px={4}
+          >
+            <Add size="1em" />
+            {'  '}
             {intl.formatMessage(this.messages['webhooks.add'])}
-          </Button>
+          </StyledButton>
         </div>
 
+        <StyledHr my={4} />
+
         {status === 'error' && <div className="error">{error}</div>}
 
         <div className="actions">
-          <Button
-            bsStyle="primary"
-            type="submit"
+          <StyledButton
+            buttonStyle="primary"
+            buttonSize="medium"
             onClick={this.handleSubmit}
             disabled={loading || !this.state.modified}
+            loading={status == 'loading'}
+            px={4}
           >
-            {submitBtnLabel}
-          </Button>
+            {captializedSubmitBtnLabel}
```

Comment:
As per https://docs.opencollective.com/help/developers/translations#dont-assume-words-order-stays-the-same-in-other-languages we shouldn't assume word's order stays the same in other languages.

We need to have a dedicated string here, something like:

```es6
<FormattedMessage
  id="webhooks.Save"
  defaultMessage="Save {count} webhooks"
  values={{count: webHookCount}}
/>
```

Reply:
@Betree  thanks a lot, can i still work on this issue

> **Code**
> 
> * Should use `styled-components`
> 
> **UI/UX**
> 
> * Webhooks aren't sent to API when saving
> * No button to remove webhook when there's only 1
> * Responsiveness issue
> 
> ![image](https://user-images.githubusercontent.com/1556356/60819799-85044680-a1a0-11e9-9e27-832a5a8a7cdf.png)